docs/modules/deployment/pages/sentinel/runtime/centos-rhel9/install-sentinel.adoc

```diff
@@ -0,0 +1,64 @@
+.Install OpenJDK 11 JRE runtime
+[source, console]
+----
+sudo dnf -y install java-11-openjdk-headless
+----
+
+ifeval::["{page-component-title}" == "Horizon"]
+include::../../../repos/rhel-centos/horizon-rhel9.adoc[]
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+include::../../../repos/rhel-centos/meridian-rhel9.adoc[]
+endif::[]
+
+.Installing the of {page-component-title} Sentinel
+[source, console]
+----
+ifeval::["{page-component-title}" == "Horizon"]
+sudo dnf -y install opennms-sentinel
+endif::[]
+ifeval::["{page-component-title}" == "Meridian"]
+sudo dnf -y install meridian-sentinel
+endif::[]
+----
+
+TIP: Disable the OpenNMS {page-component-title} repository after installation to prevent unwanted upgrades when upgrading other packages on the server.
+     After upgrade, {page-component-title} requires manual steps to upgrade configuration files or migrate database schemas to a new version.
+     For this reason, it is recommended to exclude the {page-component-title} packages from update except when you are planning on performing an upgrade.
```

Comment:
```suggestion
TIP: Disable the OpenNMS {page-component-title} repository after installation to prevent unwanted upgrades when upgrading other packages on the server.
After upgrading, {page-component-title} requires manual steps to upgrade configuration files or migrate database schemas to a new version.
For this reason, it is recommended to exclude the {page-component-title} packages from updates except when you are planning on performing an upgrade.
```